www/front_src/src/styles/partials/_side-nav.scss

```diff
@@ -13,6 +13,7 @@
   text-align: center;
   background: #e0e0e0;
   transition: all 0.2s;
+  border-right: 1px solid rgba(209, 210, 212, 1);

```

Comment:
You declare this variable : **$grey-medium** in __variables.scss_, but you don't use here ?

Isn't the seconde solution better that the first one ?
`border-right: 1px solid $grey-medium;

Reply:
done